adapters/faraday/lib/opentelemetry/adapters/faraday/adapter.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'faraday'
+require 'opentelemetry'
+
+require_relative 'middlewares/tracer_middleware'
```

Comment:
Do we have a preference for `require_relative` vs `require`? I have a mild preference for `require`, but don't want to force it on everyone unless there's more consensus.

Reply:
I tend to use `require` for resources (gems) outside of the project, and `require_relative` for internal files.

Reply:
That makes sense and appears to be an established convention. I redact my preference for require 🙂 .